jdi-light-vuetify-tests/src/main/java/io/github/com/pages/AlertsPage.java

```diff
@@ -5,16 +5,21 @@
 import com.epam.jdi.light.vuetify.elements.common.Alert;
```

Comment:
The tests on all the versions of JDK are failing. Please take a look and make it passing

Reply:
Tests are failing because not of all features are present on test site now. It'll be ok after merging and deploying all test site related PRs

Reply:
All tests are passed